kratos/containers/multi_level_data_value_container.h

```diff
@@ -0,0 +1,472 @@
+#pragma once
+
+// System includes
+#include <vector>
+
+// Kratos includes
+#include "includes/kratos_components.h"
+#include "containers/variable.h"
+
+namespace Kratos 
+{
+
+class KRATOS_API(KRATOS_CORE) MultiLevelDataValueAccesor
```

Comment:
Apart from the typo, I don't feel like _Data_ and _Value_ really add any meaning to the class' name. I've seen this recurring naming pattern in Kratos and it never gets me any closer to understanding the purpose of the class.

How about smth like `MultiLevelAccessor` or `MultiLevelContainerAccessor`? Of course other alternatives are also welcome.

Alternatively, it could just be a member class of `MultiLevelDataValueContainer` (which is suffering from the same naming pattern) and then it could just be renamed `Accessor`.

Reply:
Also, this class is very light on docs.

Reply:
Last one on this class I swear: why isn't the functionality of this class just included in the container?

Reply:
The last comment is a good point, Im not sure how easy it would be to integrate it there. I think it is up to @KratosMultiphysics/technical-committee to discuss this matter.

Reply:
we had this morning a quick discussion with @pooyan-dadvand. Indeed it would be nice to integrate in the DataValueContainer but in the short term there is no human power for doing it. We would accept this as a stopgap solution until the Container is improved (no estimate of when that would actually happen)